programs/ssl/ssl_mail_client.c

```diff
@@ -57,8 +57,8 @@
 #include <unistd.h>
 #else
 #include <io.h>
-#define read _read
-#define write _write
+#define read(fd, buf, len)  _read( fd, (void *)buf, (unsigned int)len )
+#define write(fd, buf, len) _write( fd, (const void *)buf, (unsigned int)len )
 #endif
```

Comment:
Again, I copied this from the development branch: https://github.com/ARMmbed/mbedtls/blob/development/library/net_sockets.c#L66